js/src/data/actions.js

```diff
@@ -1276,3 +1276,34 @@ export function* receiveAdsRecommendations(
 		recommendationTypes,
 	};
 }
+
+/**
+ * Action containing detailed error information.
+ *
+ * @generator
+ * @param {string} errorSlot - Unique key identifying the error (e.g., field name or error code).
+ * @param {Error|Object|null} error - The original error object or additional error details.
+ * @return {{type: string, errorSlot: string, errorType: string, error: (Error|Object|null)}}
```

Comment:
@ankitrox We are not returning the `errorType` property anymore.

Reply:
Removed it.